ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
     |     +--ro OTSi* [OTSi-carrier-id]
```

Comment:
See comment as above about using lower case identifiers.

Unless there are strong opinions on keep the OTSiG acronym, I would suggest to rename this list as:

`+--ro otsi`